docs/asciidoc/static/advanced-pipeline.asciidoc

```diff
@@ -116,6 +116,14 @@ filter {
 }
 --------------------------------------------------------------------------------
 
+To set the stdout output to json format:
+[source,json]
+--------------------------------------------------------------------------------
+output {
+    stdout {codec=>rubydebug} 
```

Comment:
OK, we can use the word json. my goal was to explain how to get the result that is shown in the example